app/src/main/java/de/mathfactory/mooltifill/AwarenessService.kt

```diff
@@ -245,7 +248,18 @@ class AwarenessService : Service() {
 
         startForeground(ONGOING_NOTIFICATION_ID, notification)
         serviceStarted.complete(Unit)
+        scheduleSendTime()
         return START_STICKY
     }
 
+    private fun scheduleSendTime() {
+        try {
+            val sendTimeRequest = OneTimeWorkRequest.Builder(SendTimeWorker::class.java).build()
```

Comment:
For a periodic task it is better to use PeriodicWorkRequest instead of repeatedly scheduling OneTimeWorkRequests.

Reply:
Thanks, @mathfactory . I updated it to using `PeriodicWorkRequest` now.

Just to share that the reason why I did the scheduling using `OneTimeWorkRequest` at the beginning was because the minimum interval for `PeriodicWorkRequest` is 15 minutes, which is hard to test.